core-ui/src/mocks/data/queryRange2.js

```diff
@@ -0,0 +1,31 @@
+import { mockGetRequest } from 'mocks/createMock';
+
+export const clusterDetailsQueryRange2 = mockGetRequest(
```

Comment:
Actually there should be only 2 queryRange mocks returning different data based on the GET req.parameters